definitions/infra-awsdatasync/golden_metrics.yml

```diff
@@ -0,0 +1,18 @@
+bytesWritten:
+  title: Bytes written
+  unit: COUNT
+  queries:
+    aws:
```

Comment:
@rahul188 are these metrics reported as Events as well? If so, you'll have to provide the event equivalent GM definition here as well. 
[Example](https://github.com/newrelic/entity-definitions/blob/main/definitions/infra-awsalb/golden_metrics.yml#L10)

Reply:
@gvassallo, i feel we are safe to move ahead with bytesWritten & filesTransferred Metrics.